docs/participant_endpoint_enums.md

```diff
@@ -0,0 +1,39 @@
+# Participant Endpoint Enums
+
+This is the list of endpoints that a PISP or DFSP must register with the switch to get callbacks from the `thirdparty-api-adapter` and `transaction-request-service`.
+
+These endpoints should be added to the enums of `central-services-shared`, and the seeds of `central-ledger`
+
+## PISP:
+
+- `THIRDPARTY_CALLBACK_URL_CONSENT_REQUEST_PUT`
+- `THIRDPARTY_CALLBACK_URL_CONSENT_REQUEST_PUT_ERROR`
+- `THIRDPARTY_CALLBACK_URL_CONSENT_POST`
+- `THIRDPARTY_CALLBACK_URL_CONSENT_REQUEST_PUT`
+- `THIRDPARTY_CALLBACK_URL_CONSENT_REQUEST_PUT_ERROR`
```

Comment:
I think you mean `CONSENT` here instead of `CONSENT_REQUEST`?

Reply:
Yes, sorry